### DISCUSSION TITLE
[type-safe-api] Add go runtime support

### DISCUSSION THREAD

**Discussion by v41e** [aws/aws-pdk#540] [Ideas]
**Open API Genertor** offers many go-server generators, as go offer better performance in benchmarks many developers prefer use go as lambda handlers, it would be awesome to have go runtime support in type-safe-api

**Comment by cogwirrel** [comment]
Agreed that would be really cool - it's quite a large undertaking to add a new runtime so it may be a while before we can implement this! Certainly open to a PR if you have the bandwidth :)

**Reply by v41e** [reply]
I think so too, just wanted to share the idea, i'll try to work on it maybe after working on a go projen project for projen community, but I think I will open PR on aws s3 integration before that's much easier ! 😂

**Reply by cogwirrel** [reply]
Haha that sounds like a good plan! Thanks!! 😁
